### ISSUE TITLE
Adjustable numbers of characters

### ISSUE THREAD

**Issue by jan-2017** [lassik/phpbb-ext-telegramnotifications#3]
I would be glad if the number of characters were adjustable, which should be displayed by the notification in telegram.bye Jan

**Comment by lassik** [comment]
This would be easy to add, but it would be even better if we could agree on an optimal number of characters for everybody :)

**Comment by jan-2017** [comment]
I have now found this setting to adjust the number. Default setting was 200 characters. But there are no notification lines displayed in the Messanger if, for example, a quote is too long. The message after the quote will probably not be shown. The user wants to read the message and not the quote content. My suggestion would be to shorten the citation length and to show only a certain number of the beginning and end of the quote.Then there is more space for the actual message.

**Comment by lassik** [comment]
I agree. I tried to remove the bbcode (the `[quote]` and other such stuff) to leave only the plain text. I used https://wiki.phpbb.com/Function.strip_bbcode but could not get it to work easily - it removed *all* of the message text, leaving an empty message :) If you know how to do it (without writing our own bbcode parser) please let me know.
